Sources/ClusterPin.swift

```diff
@@ -0,0 +1,72 @@
+import UIKit
+
+// MARK: - Pin color
+
+enum PinColor {
+  case extraSmall, small, medium, large, extraLarge
+
+  var color: UIColor {
+    switch self {
+    case .extraSmall:
```

Comment:
The idea is to have one built in `AnnotationView` with predefined styles, but it's totally up to developers to use a custom one.